data_struct.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <string.h>
+
+struct Car_type
+
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
+
+struct Car_type2
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
```

Comment:
It looks like both of these structs are identical. One of the important concepts for a struct is that you can re-use it. Instead of declaring a bunch of variables twice, you can put those variables into a struct and create two instances of the struct.

In this case, you're declaring two different struct types instead of making one struct type and reusing it. Can you update your code so that you only have one struct type that is reused for both cars?